eox_hooks/utils.py

```diff
@@ -0,0 +1,61 @@
+"""
+Utils functions for actions.
+"""
+import collections
+
+from django.conf import settings
+
+
+def flatten(d, parent_key='', sep='_'):
+    """
+    This function returns a flatten dictionary.
+    """
+    items = []
+    for k, v in d.items():
+        new_key = parent_key + sep + k if parent_key else k
+        if isinstance(v, collections.MutableMapping):
+            items.extend(flatten(v, new_key, sep=sep).items())
+        else:
+            items.append((new_key, v))
+    return dict(items)
+
+
+def _get_action_settings(action):
+    """
+    This functions returns the settings of a specific action 
+    defined inside the config dictionary, in the EOX_HOOKS_DEFINITIONS settings.
+
+    For example, if the action name is action_1, then add your configurations as follows:
+    "config" : {
+        "actions" : {
+            "action_1": {
+                {
+                    ...
+                }
+            }
+        }
+    }
+
+    Make sure the action argument matches with the key of the dict inside actions that you
+    would like to return.
+
+    Arguments:
+        - action: Name of the dictionary whose settings you want to return.  
+
+    Returns:
+        - dictionary.
+    """
+    try:
+        return settings.EOX_HOOKS_DEFINITIONS.get('config').get('actions').get(action)
```

Comment:
I think that we can remove the usage of the try block with:
```suggestion
        return settings.EOX_HOOKS_DEFINITIONS.get('config', {}).get('actions', {}).get(action, {})
```

Reply:
You are right, I changed it.